test/module/irohad/consensus/yac/yac_mocks.hpp

```diff
@@ -222,12 +223,30 @@ namespace iroha {
             hasReject, bool(PeersNumberType, PeersNumberType, PeersNumberType));
       };
 
+      class MockCleanupStrategy : public CleanupStrategy {
+       public:
+        MOCK_METHOD2(finalize,
+                     boost::optional<CleanupStrategy::RoundsType>(
+                         Round round, Answer answer));
+
+        MOCK_METHOD1(shouldCreateRound, bool(const Round &round));
+      };
+
+      std::ostream & operator<<(std::ostream &os, const CommitMessage &) {
+        return os;
+      }
+
+      std::ostream & operator<<(std::ostream &os, const RejectMessage &) {
+        return os;
+      }
```

Comment:
Are those two methods for logger? If so, did you try to `#include "logger.hpp"`? It should help to avoid defining those operators here